### DISCUSSION TITLE
What is this 'next:' key given to fetch()?

### DISCUSSION THREAD

**Discussion by harry643** [vercel/next.js#70288] [Help] [answered]
### Summary

Please can someone explain what uses this key and value?
https://github.com/vercel/next.js/blob/44d7a585ca81abfef59998a1906c85f428f08f9d/examples/cms-contentful/lib/api.ts#L45

Thank you

**Comment by Mr-Meshky** [comment, marked as answer]
This is an object that you can pass to the fetch request to control the cache or revalidation behavior.

These tags are applied to the cache. That is, it helps you to tag a group of requests with a special tag (here "posts"). This way, if you want to update or invalidate the cache related to that data (in this case, posts), it's easier.

So, you are actually saying that this request will be stored in the cache with the "posts" tag, and when the data related to the posts is changed, Next.js can manage the cache intelligently.

For example, in [rtk query](https://redux-toolkit.js.org/rtk-query/usage/automated-refetching)

**Reply by harry643** [reply]
Thanks a lot. So `next` just means Next.JS?

**Reply by harry643** [reply]
Also, for anyone else interested, I just found the feature in the Next.js docs: https://nextjs.org/docs/app/building-your-application/caching#fetch-optionsnexttags-and-revalidatetag
